cmd/disableTxOffload.go

```diff
@@ -27,17 +27,25 @@ var disableTxOffloadCmd = &cobra.Command{
 			clab.WithTimeout(timeout),
 			clab.WithRuntime(rt, debug, timeout, graceful),
 		}
-		c := clab.NewContainerLab(opts...)
-
+		c, err := clab.NewContainerLab(opts...)
+		if err != nil {
+			return err
+		}
 		ctx, cancel := context.WithCancel(context.Background())
 		defer cancel()
 
 		log.Infof("getting container '%s' information", cntName)
 
-		NSPath, err := c.Runtime.GetNSPath(ctx, cntName)
+		nodeRuntime, err := c.GetNodeRuntime(cntName)
+		if err != nil {
+			log.Fatal(err)
```

Comment:
should we return the error, rather than `log.Fatal`?

Reply:
yep, fixing that.